src/libstd/lib.rs

```diff
@@ -469,6 +469,7 @@ pub mod error;
 pub mod ffi;
 pub mod fs;
 pub mod io;
+#[cfg(not(target_os = "l4re"))]
```

Comment:
This is my main hesitation for this PR, just excluding the `net` module. This, AFAIK, doesn't happen on any other platform.